src/components/forms/PlanForm/index.tsx

```diff
@@ -705,43 +717,46 @@ const PlanForm = (props: PlanFormProps) => {
                               value={values.activities[index].actionDescription}
                             />
                           </FormGroup>
-                          <FormGroup>
-                            <Label for={`activities-${index}-goalValue`}>{QUANTITY_LABEL}</Label>
-                            <InputGroup id={`activities-${index}-goalValue-input-group`}>
-                              <Field
-                                type="number"
+                          {values.activities[index].actionCode !==
```

Comment:
Why are we hiding this field for MDAPoint?

Reply:
Actually @ciremusyoka for MDA Adverse Event(s) I am worried that completely removing the field makes things inconsistent.  Can you instead show it, but make it disabled? (i.e. not user editable)